portfolio/src/main/webapp/Home.css

```diff
@@ -0,0 +1,50 @@
+h1 {
+    font-family: "Comic Sans MS", cursive, sans-serif;
+    font-size: 50px;
+    letter-spacing: 2px;
+    word-spacing: 2px;
+    color: #ffffff;
+    font-weight: bold;
+    text-decoration: underline solid white;
+    font-style: bold;
+    font-variant: normal;
+    text-transform: none;
+    text-shadow: 0 -1px 4px #FFF, 0 -2px 10px rgb(0, 238, 255), 0 -10px 20px #008cff, 0 -18px 40px rgb(195, 0, 255);
+    text-align: center;
+}
```

Comment:
It looks like this is the same style def from Gaming.css. Maybe move the duplicates styles to a `common.css` file?